### DISCUSSION TITLE
Meta name viewport user-scalable=no

### DISCUSSION THREAD

**Discussion by laferte-tech** [quasarframework/quasar#9714] [Ideas / Proposals] [answered]
Hello,

i just noticed that on a mobile, with Chrome or Firefox, it's impossible to "pinch zoom" to enlarge pictures or text or whatever. It's working fine on Opera. I did the test on my website and Quasar.dev and i realized that the viewport is configured to;

`user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width`

I didn't pay attention to this before but user-scalable is defined to no and maximum scale to 1. I read different articles and it seems to be a bad practice (at least according to Google):
[https://web.dev/meta-viewport/](https://web.dev/meta-viewport/)

What is the reason for this? Is it about Cordova/Capacitor?

I'm just using SSR mode and i would like to know if disabling this will break some stuffs in Quasar?

Thanks a lot

**Comment by ontwikkelfabriek** [comment, marked as answer]
@GrandOurs35 

Did you try to disable it? And what happend?

Anyways I did find this Quasar zoom extension:
https://github.com/quasarframework/app-extension-qzoom

**Reply by laferte-tech** [reply]
no i didn't try, i didn't want to break everything in production :-D 
thanks for the link, but i would like to avoid another plugin.
I will give a try to change the meta and see what happen.

**Reply by laferte-tech** [reply]
for information, i tried it, i removed user-scalable and change max zoom to 5 and it seems to work nicely :)
